projboard/tests/test_client.py

```diff
@@ -111,3 +111,27 @@ def test_aboutpage(client):
     template_names = set(tmpl.origin.template_name for tmpl in response.templates)
     # And check if that 'about/about.html' is in the set.
     assert 'about/about.html' in template_names
+
+
+@pytest.mark.django_db
+def test_get_readArticlePage(client, article):
+    route = '/article/{}'.format(article.id)
+    # Send a GET request to the page
+    response = client.get(route)
+    assert response.status_code == 200
+    # Create a set of template names from the templates used in the response
+    template_names = set(tmpl.origin.template_name for tmpl in response.templates)
+    # And check if that 'about/about.html' is in the set.
+    assert 'show/read_article.html' in template_names
+
+
+@pytest.mark.django_db
+def test_post_readArticlePage(client, article):
```

Comment:
same here

Reply:
This tests created get request, not post, 
It seems as duplicate test with different name

Reply:
done